examples/process-persistence/README.md

```diff
@@ -109,18 +93,31 @@ This is the maven profile for Postgresql
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-postgresql</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-container-image-jib</artifactId>
+        </dependency>
     </dependencies>
 </profile>
 ```
-The only dependency needed is postgresql jdbc driver.
+The dependency needed are postgresql jdbc driver and `quarkus-container-image-jib`.
+The `quarkus-container-image-jib` is an extension for the Quarkus framework that enables
+building container images using Jib and helps to build the optimized container images directly.
 
-This is the configuration for Postgresql
+These are the configurations for Postgresql
 ```
 %postgresql.quarkus.datasource.db-kind=postgresql
-```
 
-When running the example in Dev mode, Quarkus will stat a `Postgresql` database container as a part of the Dev
-Services. So make sure to install docker before running this example.
+%postgresql.quarkus.container-image.build=true
+%postgresql.quarkus.container-image.push=false
+%postgresql.quarkus.container-image.group=bamoe
+%postgresql.quarkus.container-image.registry=dev.local
+%postgresql.quarkus.container-image.tag=${project.version}
+%postgresql.quarkus.container-image.name=process-persistence-postgresql
+```
+The `%postgresql.quarkus.datasource.db-kind` configuration property is used to explicitly specify the type of database to connect.
+Different properties configured using the `postgresql.quarkus.container-image` define image-related settings like
+group, registry, tag & name needed to build the image.
 
```

Comment:
You could mention the container specific properties in a section of its own after the database specific settings.